monitor/checkWorker.js

```diff
@@ -63,6 +65,7 @@ async function checkWorker() {
     vBalances.ok = vBalances.homeOk && vBalances.foreignOk
     vBalances.health = true
     writeFile(`/responses/${MONITOR_BRIDGE_NAME}/validators.json`, vBalances)
+    saveCache()
```

Comment:
I don't see the reason why it will be better. I think it makes sense to save the caches on disk at the very end of the script, so that we don't lose any changes in case transaction senders will be fetched in other functions as well. Even though now it is not the case, it can be in the future.

Reply:
In general you are right. But in this particular case we request the state of the validators contract in the line 41
```
const vBalances = await validators(bridgeMode)
```
so, if the script fails here due to some exception the cache will not be stored.

Reply:
Ok, that makes some sense to me, let's do it this way.